.github/workflows/selenium-e2e.yml

```diff
@@ -1,32 +0,0 @@
-# This is a basic workflow that is manually triggered
```

Comment:
I don't know that you want to remove this file since it's already in git. I think you just want to not to any updates to it in this PR.